### ISSUE TITLE
Is It Possible For A Steamdeck/Linux Port

### ISSUE THREAD

**Issue by Tmedina923** [ZenoArrows/The-Simpsons-Hit-and-Run#51]

**Comment by Tmedina923** [comment]
I saw that this was ported to PS Vita and Nintendo Switch, maybe Steamdeck could also be included if possible since that's also a popular portable device as well 🤷

**Comment by ZenoArrows** [comment]
Possible yeah, but you'd lose the modding support so it doesn't seem desirable.

**Comment by OmniBlade** [comment]
I'd also be interested in seeing native 64bit Linux (and maybe even Windows?) builds for the engine. I'm not familiar with this games modding scene, but I assume current mods for the game are achieved by using some kind of code injection into the original binary to add new functionality, otherwise I would expect data only mods to work with a rebuilt version of the game regardless? With that in mind, couldn't this version of the engine include its own implementations of whatever engine extensions the community had developed?

**Comment by OmniBlade** [comment]
So I tried compiling on linux for the heck of it and it compiles correctly it seems. With data from retail I can get it to show the copyright screen or even the language select with the PAL options built in. However it segfaults in radMoviePlayer::Load trying to play a video? Could be it doesn't like the version of ffmpeg on Ubuntu I guess?

**Comment by OmniBlade** [comment]
After reading the issue that "corrupt" videos designed to make the original skip them cause crashes in the video player I replaced them in my data and managed to get to loading the first mission. segfaults in CGuiScreenPauseSettings::CGuiScreenPauseSettings.

**Comment by OmniBlade** [comment]
Also, it seems debug builds don't work for me, they trigger asserts, have you experienced that @ZenoArrows

**Comment by ZenoArrows** [comment]
@OmniBlade You need to use the art assets from either the Switch or Vita releases, due to changes to support consoles the game is no longer compatible with the UI assets from the PC release.

**Comment by OmniBlade** [comment]
Yeah, that has got me in game now. I managed to complete first mission and got a segfault during the second. Will test further when I have more time.
